trin-storage/src/utils.rs

```diff
@@ -26,9 +20,11 @@ pub fn setup_sql(node_data_dir: &Path) -> Result<Pool<SqliteConnectionManager>,
     let manager = SqliteConnectionManager::file(sql_path);
     let pool = Pool::new(manager)?;
     let conn = pool.get()?;
-    conn.execute_batch(CREATE_QUERY_DB)?;
+    conn.execute_batch(CREATE_QUERY_DB_HISTORY)?;
```

Comment:
I think it's safe to move `CREATE_QUERY_DB_HISTORY` to `HistoryStorage` and run it only when it's created.
Similarly, maybe we can move Beacon and LcUpdate to `BeaconStorage` as well.

I'm not sure about `STORE_INFO_CREATE_TABLE`, as that migh have to run multiple times and there isn't a constructor than we can easily use. Maybe there should be...